CamperBot.md

```diff
@@ -10,7 +10,7 @@ Hi, I'm **[CamperBot](https://github.com/FreeCodeCamp/freecodecamp/wiki/camperbo
 - `about @username` shows info on that user
 - `Algorithm BONFIRENAME` info on a Algorithm
 
-meet CamperBot in [![FreeCodeCamp/camperbotPlayground](https://img.shields.io/badge/Gitter_Chat_Room:-FreeCodeCamp/camperbotPlayground_%E2%86%91-006400.svg?style=flat-square&maxAge=2592000%29.svg)](https://gitter.im/FreeCodeCamp/camperbotPlayground)
+meet CamperBot in ![FreeCodeCamp/camperbotPlayground](https://img.shields.io/badge/Gitter_Chat_Room:-FreeCodeCamp/camperbotPlayground_%E2%86%91-006400.svg?style=flat-square&maxAge=2592000%29.svg)
```

Comment:
It was not meant to, I used a regex and it was easier to check which one I had to undo after making the PR.